docs/installation_guide.md

```diff
@@ -302,110 +348,118 @@ $ sudo docker-compose -f ./docker-compose.yml -f ./docker-compose.notary.yml -f
 Please check the [Docker Compose command-line reference](https://docs.docker.com/compose/reference/) for more on docker-compose.
 
 ### Persistent data and log files
+
 By default, registry data is persisted in the host's `/data/` directory.  This data remains unchanged even when Harbor's containers are removed and/or recreated.  
 
-In addition, Harbor uses *rsyslog* to collect the logs of each container. By default, these log files are stored in the directory `/var/log/harbor/` on the target host for troubleshooting.  
+In addition, Harbor uses [rsyslog](https://www.rsyslog.com/) to collect the logs from each container. By default, these log files are stored in the `/var/log/harbor/` directory on the target host for troubleshooting.  
 
-## Configuring Harbor listening on a customized port
-By default, Harbor listens on port 80(HTTP) and 443(HTTPS, if configured) for both admin portal and docker commands, you can configure it with a customized one.  
+## Configuring Harbor to listen on a customized port
 
-### For HTTP protocol
+By default, Harbor listens on ports 80 (HTTP) and 443 (HTTPS, if configured) for both admin portal and Docker commands. But you can also configure it to listen on custom ports. 
 
-1.Modify docker-compose.yml  
-Replace the first "80" to a customized port, e.g. 8888:80.  
+### HTTP
 
-```
-proxy:
+1. Modify `docker-compose.yml` and replace the first `80` with a custom port, e.g. `8888:80`.
+
+    ```yaml
```

Comment:
Double-check the indentation for this block. Some entries need two more spaces, some have two spaces too many.